ddtrace/tracer/span.go

```diff
@@ -369,9 +369,16 @@ func (s *span) finish(finishTime int64) {
 	s.context.finish()
 }
 
+func normalizeSpan(s *span) {
```

Comment:
Do we need a separate function? Let's just inline this.

Reply:
We don't need this function anymore :)